src/services/user.js

```diff
@@ -18,6 +18,7 @@ const {
   roles: { ADMIN }
 } = require('~/consts/auth')
 const { allowedTutorFieldsForUpdate } = require('~/validation/services/user')
+const { allowedStudentFieldsForUpdate } = require('~/validation/services/user')

```

Comment:
It would be nice to import both allowed fields consts in single line
```suggestion
const { allowedTutorFieldsForUpdate, allowedStudentFieldsForUpdate } = require('~/validation/services/user')

```